CHANGES.rst

```diff
@@ -1159,6 +1159,8 @@ Other Changes and Additions
 - ``extract_array`` raises a ``ValueError`` if the data type of the
   input array is inconsistent with the ``fill_value``. [#10602]
 
+- Add Azure Pipelines configuration for auto-releasing source and
```

Comment:
It's not clear to me this is needed, since this is about the CI infrastructure and docs, which don't normally require changelog entries?

Reply:
Ok I'll remove